proto/osmosis/incentives/gauge.proto

```diff
@@ -50,6 +50,14 @@ message Gauge {
   ];
 }
 
+// Gauge is an object that stores GroupGaugeId as well as internalGaugeIds. We
+// linked these two together so that we can distribute tokens from groupGauge to
+// internalGauges.
+message GroupGauge {
+  uint64 group_gauge_id = 1;
+  repeated uint64 internal_ids = 2;
```

Comment:
We should include a splitting policy field here as an enum, even if it currently just evenly splits

Reply:
added: https://github.com/osmosis-labs/osmosis/pull/6085/files#diff-ba512501fa081ff95a2b23175b9bba1ef438ff455e357e71ce884da0add490d5R54